src/transformState.js

```diff
@@ -62,7 +62,24 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    switch (transform['operation']) {
+      case 'addProperties':
+        for (const key in transform['properties']) {
+          state[key] = transform['properties'][key];
+        }
+        break;
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
```

Comment:
```suggestion
      case 'clear':
        state = {};
```

Reply:
Linter error

Assignment to function parameter 'state'  no-param-reassign

Reply:
Sorry, my bad